pubspec.lock

```diff
@@ -408,4 +415,4 @@ packages:
     source: hosted
     version: "2.1.13"
 sdks:
-  dart: ">=1.23.0 <=2.0.0-dev.15.0"
+  dart: ">=1.23.0 <=2.0.0-edge.f5580e540847b411bd10b6836a9f62b1c27f076f"
```

Comment:
not clear to me that this matters a lot, but perhaps leaving this at a dev release? (`2.0.0-dev.15.0` / `2.0.0-dev.16.0`)

Reply:
done